### PR TITLE
Monitoring Documentation

### LINKED ISSUE #363: Documentation required for monitoring.
## Is this a BUG REPORT or FEATURE REQUEST?
 
FEATURE REQUEST

**Feature**: Documentation required for chaos monitoring, and interleaving Grafana dashboards with chaos events and chaos exporter metrics.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
**What this PR does / why we need it**: This PR adds docs for the monitoring aspect of LitmusChaos.

**Which issue this PR fixes**: fixes #363 

**Special notes for your reviewer**: None.

**Checklist:**
-   [x] Fixes #363 
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/monitoring.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ sidebar_label: Monitoring
...
 
-Building a hypothesis around steady state behaviour, varying real world events, running experiments in production and automating them to run as a workflow in CI pipelines and minimizing the blast radius are some advanced chaos practices which are all backed by an extensive monitoring infrastructure managed by SREs heading IT operations. Monitoring chaos and performance metrics is a observability paradigm providing real-time insights into the four golden signals for monitoring distributed systems namely, latency, traffic, errors, and saturation.
+Building a hypothesis around steady-state behaviour, varying real-world events, running experiments in production and automating them to run as a workflow in CI pipelines and minimizing the blast radius are some advanced chaos practices which are all backed by an extensive monitoring infrastructure managed by SREs heading IT operations. Monitoring chaos and performance metrics is an observability paradigm providing real-time insights into the four golden signals for monitoring distributed systems namely, latency, traffic, errors, and saturation.
```
Comment (divya-mohan0209) [inline docs/monitoring.md]: Possible reword to: "Building a hypothesis around steady-state behaviour, varying real-world events, running experiments in production, automating them to run as a workflow in CI pipelines, and minimizing the blast radius are some advanced chaos practices. These are all backed by extensive monitoring infrastructure managed by SREs heading IT operations. Monitoring chaos and performance metrics is an observability paradigm providing real-time insights into the four golden signals for monitoring distributed systems namely, latency, traffic, errors, and saturation."

**Comment by ishangupta-ds** [comment]
@ksatchit @umamukkara @divya-mohan0209 @ajeshbaby Please take a look.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Minor rewording comments added.

File: docs/monitoring.md (hunk 1 above)
Comment (ishangupta-ds) [reply docs/monitoring.md]: surely.. @divya-mohan0209

**Comment by ishangupta-ds** [comment]
@divya-mohan0209 @ksatchit PTAL.

**Review by k8s-dev — APPROVED** [review]
LGTM.

**Review by ksatchit — APPROVED** [review]

**Pull request merged**